ci-operator/jobs/openshift/release/openshift-release-periodics.yaml

```diff
@@ -1,4 +1,87 @@
 periodics:
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 8h
+  name: release-promote-openshift-machine-os-content-e2e-aws-4.0
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --secret-dir=/usr/local/e2e-aws-cluster-profile
+      - --template=/usr/local/e2e-aws
```

Comment:
The installer is still hitting the `builds.json` API to decide which RHCOS build to use for the AMIs, so I dunno if this promotion test is actually protecting us from boot-image issues at the moment.  For example:

```console
$ curl -s https://storage.googleapis.com/origin-ci-test/logs/release-promote-openshift-machine-os-content-e2e-aws-4.0/10/artifacts/e2e-aws/installer/.openshift_install.log | grep RHCOS
time="2019-03-11T03:28:56Z" level=debug msg="Fetching RHCOS builds from \"https://releases-rhcos.svc.ci.openshift.org/storage/releases/maipo/builds.json\""
time="2019-03-11T03:28:57Z" level=debug msg="Fetching RHCOS metadata from \"https://releases-rhcos.svc.ci.openshift.org/storage/releases/maipo/400.7.20190306.0/meta.json\""
```

We'll also need something like the in-flight openshift/installer#1286 and maybe openshift/origin#21998 for that.

I guess this promotion test is currently giving us gate-protection for the OSContainer image, protecting us from post-pivot changes?